internal/website/sidebars.js

```diff
@@ -177,9 +177,35 @@ module.exports = {
       id: 'gqty-intro',
     },
     {
-      type: 'doc',
-      label: 'Release Notes',
-      id: 'release-notes',
```

Comment:
I wonder if we want to keep a page named `release-notes` with links to the appropriate changelogs and archive for SEO purposes/preserve the link in case anyone lands on it from Discord, etc.

Reply:
Yeah, that's a good call. I just moved the `/docs/changelog/archive` page back to `/docs/release-notes`. The content of that page already included the new location links at the top, so that worked out well. Organization of the site isn't impacted since the sidebar is defined separately from the page structure. I briefly considered setting up a redirect from `/docs/release-notes` to `/docs/changelog/archive` instead, but that seemed unnecessarily complicated.

[Preview](https://hcixzyt38dn5ak04xxcqc36lf.js.wpenginepowered.com/docs/release-notes)

Reply:
Awesome! Looks great